research/2022-12-05-Autofill_feature.md

```diff
@@ -0,0 +1,25 @@
+# Autofill Feature Usability Testing
+
+## Key Headlines
+
+- The question type overview page is visually easy for users to engage with but for some it is just on the edge of being too long and some believe it is important to be able to see all the questions on one page without scrolling down.
```

Comment:
Not sure this needs to include "overview" in the "question type overview page" as an overview is different to the page in question

Reply:
Good point, removed that word.